example/trigger/src/main/java/org/apache/iotdb/trigger/AlertingTriggerExample.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.trigger;
+
+import org.apache.iotdb.db.engine.trigger.api.Trigger;
+import org.apache.iotdb.db.engine.trigger.api.TriggerAttributes;
+import org.apache.iotdb.db.sink.alertmanager.AlertManagerConfiguration;
+import org.apache.iotdb.db.sink.alertmanager.AlertManagerEvent;
+import org.apache.iotdb.db.sink.alertmanager.AlertManagerHandler;
+
+import java.util.HashMap;
+
+public class AlertingTriggerExample implements Trigger {
```

Comment:
Considering that there is already a `TriggerExample` under this package, I suggest to change the name of `AlertingTriggerExample` (or change the name of `TriggerExample`)